From bad354c90d371ab65efb7a60b8c153a521fa6d87 Mon Sep 17 00:00:00 2001
From: Liu Xiaowen <b37945@freescale.com>
Date: Fri, 13 Dec 2013 11:26:10 +0800
Subject: [PATCH 270/602] ENGR00215174 add PREBUILT_FSL_WFDSINK to control
 wfdsink build.

add PREBUILT_FSL_WFDSINK to control wfdsink build.

Signed-off-by: Liu Xiaowen <b37945@freescale.com>
---
 imx6/BoardConfigCommon.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/imx6/BoardConfigCommon.mk b/imx6/BoardConfigCommon.mk
index 4aa434f..96dbb75 100644
--- a/imx6/BoardConfigCommon.mk
+++ b/imx6/BoardConfigCommon.mk
@@ -48,6 +48,7 @@ UMS_ONEPARTITION_PER_DISK := true
 PREBUILT_FSL_IMX_CODEC := true
 PREBUILT_FSL_IMX_OMX := false
 PREBUILT_FSL_IMX_GPU := true
+PREBUILT_FSL_WFDSINK := true
 
 # use non-neon memory copy on mx6x to get better performance
 ARCH_ARM_USE_NON_NEON_MEMCPY := true
-- 
1.8.0

